tests/sentry/api/endpoints/test_organization_integration_issues.py

```diff
@@ -1,10 +1,10 @@
 from unittest.mock import patch
 
 from sentry.testutils import APITestCase
-from sentry.testutils.silo import region_silo_test
+from sentry.testutils.silo import control_silo_test
 
 
-@region_silo_test(stable=True)
+@control_silo_test(stable=True)
```

Comment:
Do you think we'll need to move ExternalIssues to ControlSilo as well?

Reply:
Yeah, I agre, I'm backing up on this and looking for the right path forward.